Contribution page shows deprecated Poetry installation instructions

The contributing.rst file shows the usage of curl piped to a python interpreter, while pipx is the preferred method of installation.

I then tested the setup by running the vpype tests and had 4 fail.  
```poetry run pytest
========================================================== test session starts ==========================================================
platform linux -- Python 3.8.10, pytest-7.1.2, pluggy-1.0.0
Matplotlib: 3.5.2
Freetype: 2.6.1
benchmark: 3.4.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /home/mike/Documents/Projects/plotters/vpype, configfile: pyproject.toml
plugins: mpl-0.16.0, benchmark-3.4.1, cov-3.0.0
collected 1289 items / 4 errors                                                                                                         

================================================================ ERRORS =================================================================
_______________________________________________ ERROR collecting tests/test_benchmarks.py _______________________________________________
ImportError while importing test module '/home/mike/Documents/Projects/plotters/vpype/tests/test_benchmarks.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_benchmarks.py:7: in <module>
    from vpype_viewer import ViewMode, render_image
vpype_viewer/__init__.py:6: in <module>
    from .engine import *
vpype_viewer/engine.py:10: in <module>
    import moderngl as mgl
E   ModuleNotFoundError: No module named 'moderngl'
__________________________________________________ ERROR collecting tests/test_show.py __________________________________________________
ImportError while importing test module '/home/mike/Documents/Projects/plotters/vpype/tests/test_show.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_show.py:9: in <module>
    import vpype_viewer
vpype_viewer/__init__.py:6: in <module>
    from .engine import *
vpype_viewer/engine.py:10: in <module>
    import moderngl as mgl
E   ModuleNotFoundError: No module named 'moderngl'
------------------------------------------------------------ Captured stderr ------------------------------------------------------------
WARNING:matplotlib.font_manager:Matplotlib is building the font cache; this may take a moment.
__________________________________________________ ERROR collecting tests/test_text.py __________________________________________________
ImportError while importing test module '/home/mike/Documents/Projects/plotters/vpype/tests/test_text.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_text.py:6: in <module>
    from vpype_viewer import ImageRenderer, UnitType
vpype_viewer/__init__.py:6: in <module>
    from .engine import *
vpype_viewer/engine.py:10: in <module>
    import moderngl as mgl
E   ModuleNotFoundError: No module named 'moderngl'
_________________________________________________ ERROR collecting tests/test_viewer.py _________________________________________________
ImportError while importing test module '/home/mike/Documents/Projects/plotters/vpype/tests/test_viewer.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_viewer.py:8: in <module>
    from vpype_viewer import (
vpype_viewer/__init__.py:6: in <module>
    from .engine import *
vpype_viewer/engine.py:10: in <module>
    import moderngl as mgl
E   ModuleNotFoundError: No module named 'moderngl'
======================================================== short test summary info ========================================================
ERROR tests/test_benchmarks.py
ERROR tests/test_show.py
ERROR tests/test_text.py
ERROR tests/test_viewer.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 4 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================== 4 errors in 12.21s ===========================================================
```
I then manually installed `moderngl` using pip and retried, then repeated the process after installing `PySide2` before the tests finally ran and passed successfully.
I'm unsure whether these packages are supposed to be installed as part of the Poetry install.

--- a/docs/contributing.rst
+++ b/docs/contributing.rst
@@ -37,17 +37,22 @@ Development environment
 .. highlight:: bash
 
 *vpype* uses `Poetry <https://python-poetry.org>`_ for packaging and dependency management and its `installation
-<https://python-poetry.org/docs/#installation>`_ is required to prepare the development environment::
+<https://python-poetry.org/docs/#installation>`_ is required to prepare the development environment. It can be
+installed either using an install script, or using pipx. Run this command to use the install script::
 
-  $ curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+  $ curl -sSL https://install.python-poetry.org | python3 -
+  
+Run this command for a pipx install::
 
-(See Poetry's documentation for alternative means of installation.)
+  $ pipx install poetry
+
+See Poetry's `installation instructions <https://python-poetry.org/docs/#installation>`__ for more information.
 
 You can then download *vpype*, prepare a virtual environment and install all dependencies with a few commands::
 
   $ git clone https://github.com/abey79/vpype.git
   $ cd vpype
-  $ poetry install
+  $ poetry install -E all
 
 You can execute *vpype* (which installed in the project's virtual environment managed by Poetry) with the ``poetry
 run`` command::

